editor/src/clj/editor/ui.clj

```diff
@@ -1324,7 +1324,7 @@
                                 label
                                 icon
                                 style-classes
-                                (make-menu-items scene options command-contexts {} evaluation-context)
+                                (make-menu-items scene options command-contexts command->shortcut evaluation-context)
```

Comment:
Not sure why we didn't use `command->shortcut` in the submenus, but without that we didn't show shortcuts in nested submenus.